src/libraries/System.Security.Cryptography/tests/System.Security.Cryptography.Tests.csproj

```diff
@@ -14,8 +14,6 @@
     <TargetPlatformIdentifier>$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)'))</TargetPlatformIdentifier>
   </PropertyGroup>
   <PropertyGroup Condition="'$(TargetOS)' == 'Browser'">
-    <HelixTargetsFile>wasm.helix.targets</HelixTargetsFile>
```

Comment:
Can we delete the wasm.helix.targets file as well?

Reply:
I moved the file to eng/testing as a sample.